docs/reference.md

```diff
@@ -32,6 +31,11 @@ Deprecated:
 
 - `asMap` - Renamed to [`asObject`](#asObject).
 
+Deprecated:
+
+- `asMap` - Deprecated alias for `asObject`.
+- `asNone` - Use `asOptional(asValue(null))` instead.
+
```

Comment:
Must have been a merge conflict, but there are two lists of deprecated things now.